python/tk_multi_publish2/dialog.py

```diff
@@ -521,13 +531,19 @@ def _update_item_thumbnail(self, pixmap):
             self._summary_thumbnail = pixmap
             if pixmap:
                 # update all items with the summary thumbnail
-                for top_level_item in self._plugin_manager.top_level_items:
+                top_level_items = list(self._publish_manager.tree.root_item.children)
+                for top_level_item in top_level_items:
                     top_level_item.thumbnail = self._summary_thumbnail
                     top_level_item.thumbnail_explicit = False
-                    top_level_item._propagate_thumbnail_to_children()
```

Comment:
Eliminated the need to call a private method of an item here by just injecting the logic it was using.